BraveShared/BraveStrings.swift

```diff
@@ -2004,6 +2004,41 @@ extension Strings {
             NSLocalizedString("sync.v2MigrationErrorMessage", tableName: "BraveShared", bundle: .braveShared,
                               value: "Failed to migrate bookmarks. Please try again later.",
                               comment: "Message for popup when the bookmark migration fails")
+        /// History Migration localization text
+        public static let historyMigrationErrorMessage =
+            NSLocalizedString("sync.historyMigrationErrorMessage", tableName: "BraveShared", bundle: .braveShared,
+                              value: "Failed to migrate hitory. Please try again later.",
+                              comment: "Message for popup when the history migration fails")
+        public static let syncConfigurationInformationText =
+            NSLocalizedString("sync.syncConfigurationInformationText", tableName: "BraveShared", bundle: .braveShared,
+                              value: "Manage what information you would like to sync between devices. These settings only affect this device.",
+                              comment: "Information Text underneath the toggles for enable/disable different sync types for the device")
+        public static let syncSettingsTitle =
+            NSLocalizedString("sync.syncConfigurationInformationText", tableName: "BraveShared", bundle: .braveShared,
```

Comment:
Duplicate key `sync.syncConfigurationInformationText` for both `syncConfigurationInformationText` and `syncSettingsTitle`

Reply:
Fix at https://github.com/brave/brave-ios/pull/3703/commits/78c29884f6afce95aa286694b9316fb58c33734f